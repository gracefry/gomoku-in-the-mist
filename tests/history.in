place S1
place L2
place A3
place G4
history
term
