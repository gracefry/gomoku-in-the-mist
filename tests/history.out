S1L2A3G4
